tests/integration/telemetry/outboundtrafficpolicy/helper.go

```diff
@@ -374,7 +374,6 @@ func setupEcho(t *testing.T, ctx resource.Context, mode TrafficPolicy) (echo.Ins
 			},
 			TLSSettings: &common.TLSSettings{
 				// Echo has these test certs baked into the docker image
-				RootCert:   mustReadCert(t, "cacert.pem"),
```

Comment:
Why is this removed? This is for the test image (dockerfile.app). I don't see why it should not have this (but keep the cert.crt and key?).

For pilot it clearly shouldn't, but the app image is meant exclusively for tests

Reply:
it's replaced to also use the system cert pool just like the change in pilot, I don't know what is the reason for using cacert.pem even in test? It's just a forked version of the CA certs.

cert.crt is a self-signed cert and has nothing to do with cacert.pem but I'm not sure how it is used in the test.

Reply:
I see. Well it seems clear the test is wrong then, maybe we use InsecureSkipVerify somewhere